static/src/stylesheets/components/sass-mq/_mq.scss

```diff
@@ -33,6 +33,7 @@ $mq-responsive: true !default;
 /// @link https://github.com/sass-mq/sass-mq#seeing-the-currently-active-breakpoint Full documentation and examples
 $mq-breakpoints: (
     mobile:  320px,
+    phablet: 620px,
```

Comment:
Actually that breakpoint is a different size: I need 620px rather than 660px. Is it ok to define a new 620px breakpoint with a non-conflicting name in my own cs file (embed.css)?